frontend/package.json

```diff
@@ -66,6 +66,7 @@
     "js-yaml": "3.x",
     "lodash-es": "4.x",
     "patternfly": "^3.45.0",
+    "patternfly-react": "^2.11.2",
```

Comment:
We should go to 2.12, that will reduce the size cost of including pf-react since with that release it no longer pulls in all of react-bootstrap (only the components actually used).

Reply:
+1 for not importing the whole world!